t/mojolicious/perl_config_lite_app.t

```diff
@@ -0,0 +1,21 @@
+use Mojo::Base -strict;
+
+BEGIN {
+  $ENV{MOJO_MODE}    = 'development';
+  $ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll';
+}
```

Comment:
Why is that necessary for the test?

Reply:
Cargo-culted from `json_config_lite_app.t`; I removed it.